engines/director/detection_tables.h

```diff
@@ -8894,6 +8926,21 @@ static const DirectorGameDescription gameDescriptions[] = {
 
 	WINGAME1t_l("onsalakorv", "", "onsala.EXE", "41d87422d7c0f48886a99a7fe05ba1c0", 1713409, Common::SV_SWE, 650),
 
+	WINGAME1("oscar3", "",   "ossee32.exe", "t:c4953efef8d157b15a1220bb2984da4b", 1855956, 601),
+	WINGAME1("oscar3", "",   "ossee16.exe", "t:ea00ef6376eacdb8d2fce1beb4205443", 1394306, 601),
+
+	MACGAME1("oscar3", "",   "Oscar taucht unter",	"t:4288b31a968989b4918c8696e0a7324f", 1535536, 601),
+
+	WINGAME2t("oscar4", "",  "start32.exe", "t:6eeb631d693362c48f73daf08fe03e1e", 1661334,
+							 "Oscar4.ico", "23f8d1b556b0134b792ec3740ebd95de", 766, 601),
```

Comment:
Aren't there any director files to use instead of the icon? And you can also use `t:` in the checksum and `WINGAME2`